tests/baselines/reference/ES5SymbolProperty2.errors.txt

```diff
@@ -1,15 +1,12 @@
-tests/cases/conformance/Symbols/ES5SymbolProperty2.ts(5,10): error TS2471: A computed property name of the form 'Symbol.iterator' must be of type 'symbol'.
 tests/cases/conformance/Symbols/ES5SymbolProperty2.ts(10,11): error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the `lib` compiler option to es2015 or later.
 
 
-==== tests/cases/conformance/Symbols/ES5SymbolProperty2.ts (2 errors) ====
+==== tests/cases/conformance/Symbols/ES5SymbolProperty2.ts (1 errors) ====
     module M {
         var Symbol: any;
     
         export class C {
             [Symbol.iterator]() { }
-             ~~~~~~~~~~~~~~~
-!!! error TS2471: A computed property name of the form 'Symbol.iterator' must be of type 'symbol'.

```

Comment:
This used to be close to giving a good error. Do we have any strictness flag that disallows this?

Reply:
Nope? Pretty sure we allow computed named with completely arbitrary (key compatible) types and just infer an index signature.

Reply:
Yeah, but what do you want to do when you have a key of type `any`? I can't imagine why you'd want that behavior under `--strict`.

Reply:
I mean, when `Symbol` isn't typed `any`, as in this test, we issue a lookup error...